sale_stock_picking_invoicing/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2013-Today - Akretion (<http://www.akretion.com>).
+# @author Renato Lima <renato.lima@akretion.com.br>
+# @author Raphael Valyi <raphael.valyi@akretion.com>
+# @author Magno Costa <magno.costa@akretion.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sales Stock Picking Invocing",
+    "category": "Warehouse Management",
+    "license": "AGPL-3",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "version": "14.0.1.0.0",
+    "maintainers": ["mbcosta", "renatonlima"],
+    "depends": [
+        "sale_management",
```

Comment:
are you sure sale_management is required? If yes could you please just explain why? (and eventually leave a short comment why here)

Reply:
Necessary to access Sale Menu
![image](https://github.com/OCA/account-invoicing/assets/6341149/f0528072-199b-41b1-bd79-2b213e947977)

![image](https://github.com/OCA/account-invoicing/assets/6341149/fe4de512-6ef3-4a35-aec7-05f6e073d56f)

By some reason the sale module create menuitem with attribute active="False" https://github.com/OCA/OCB/blob/14.0/addons/sale/views/sale_views.xml#L8 and sale_management change for active="True" https://github.com/OCA/OCB/blob/14.0/addons/sale_management/views/sale_management_views.xml#L4